Managers/AudioMan.cpp

```diff
@@ -1,1519 +1,972 @@
-//////////////////////////////////////////////////////////////////////////////////////////
-// File:            AudioMan.cpp
-//////////////////////////////////////////////////////////////////////////////////////////
-// Description:     Source file for the AudioMan class.
-// Project:         Retro Terrain Engine
-// Author(s):       Daniel Tabar
-//                  data@datarealms.com
-//                  http://www.datarealms.com
-
-
-//////////////////////////////////////////////////////////////////////////////////////////
-// Inclusions of header files
-
-#ifdef __USE_SOUND_FMOD
-#include "fmod.h"
-#elif __USE_SOUND_SDLMIXER
-#include "SDL.h"
-#include "SDL_mixer.h"
-#elif __USE_SOUND_GORILLA
-#include "gorilla/ga.h"
-#include "gorilla/gau.h"
-#endif
-
 #include "AudioMan.h"
 #include "ConsoleMan.h"
 #include "SettingsMan.h"
 #include "Sound.h"
 #include "DDTError.h"
 
-// Allegro defines those via define in astdlib.h and Boost with stdlib go crazy about those so we need to undefine them manually.
-#undef int8_t
-#undef uint8_t
-#undef int16_t
-#undef uint16_t
-#undef int32_t
-#undef uint32_t
-#undef intptr_t
-#undef uintptr_t
-
-#undef LONG_LONG
-#undef int64_t
-#undef uint64_t
-
-#ifdef __USE_SOUND_FMOD
-#define MAX_VOLUME 255
-#elif __USE_SOUND_SDLMIXER
-#define MAX_VOLUME MIX_MAX_VOLUME
-#elif __USE_SOUND_GORILLA
-#define MAX_VOLUME 1.0f
-#endif
+namespace RTE {
+	const std::string AudioMan::m_ClassName = "AudioMan";
 
-namespace RTE
-{
+	// I know this is a crime, but if I include it in FrameMan.h the whole thing will collapse due to int redefinitions in Allegro
+	std::mutex g_SoundEventsListMutex[c_MaxClients];
 
-const string AudioMan::m_ClassName = "AudioMan";
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+	// Callback for catching music streams that end
```

Comment:
Spacing here seems wonky

Reply:
It's not, it's one tab in like the function under it.
It isn't a comment on the preprocessor conditional.

Reply:
Oops!